pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -1519,11 +1519,18 @@ LogicalResult ConvertAtenOp<ValueTensorLiteralOp>::matchAndRewrite(
     ValueTensorLiteralOp op,
     OpAdaptor adaptor,
     ConversionPatternRewriter& rewriter) const {
-  auto outputTy = getTypeConverter()
-                      ->convertType(op.getType())
-                      .template cast<RankedTensorType>();
-  rewriter.replaceOpWithNewOp<mhlo::ConstOp>(op, outputTy, adaptor.value());
-
+  MLIRContext* context = op->getContext();
+  if (auto elements = op.valueAttr().dyn_cast<DenseIntElementsAttr>()) {
```

Comment:
Why do we need to treat int attribute specially?

Reply:
It seems `mhlo.constant` does not support `sint` type, so I should deal with INT especially.

``` text
error: 'mhlo.constant' op result #0 must be statically shaped tensor of floating-point or pred (AKA boolean or 1-bit integer) or 8/16/32/64-bit signless integer or 8/16/32/64-bit unsigned integer or complex type with 32-bit float or 64-bit float elements values, but got 'tensor<4xsi64>'
// -----// IR Dump After ConvertTorchToMhlo Failed //----- //
"func.func"() ({
  %0 = "mhlo.constant"() {value = dense<[1, 2, 3, 4]> : tensor<4xsi64>} : () -> tensor<4xsi64>
  %1 = "torch_c.from_builtin_tensor"(%0) : (tensor<4xsi64>) -> !torch.vtensor<[4],si64>
  "func.return"(%1) : (!torch.vtensor<[4],si64>) -> ()
}) {arg_attrs = [], function_type = () -> !torch.vtensor<[4],si64>, sym_name = "main"} : () -> ()
```